workflowhub_graph/workflowhub_graph/absolutize.py

```diff
@@ -48,7 +48,7 @@ def make_paths_absolute(
     json_data = copy.deepcopy(json_data)
 
     workflow_url = (
-        f"{workflowhub_url}/workflows/{workflow_id}/ro_crate?version={workflow_version}"
+        f"{workflowhub_base_url}/workflows/{workflow_id}"
```

Comment:
As discussed on Teams, the pattern below will result in resolvable URIs (based on some manual testing). They won't resolve to the exact files but they will yield the downloadable zipped crate in which the files can be found.

This also has the benefit of maintaining unique base URIs for each version of a workflow, though I am not sure if all versions are included in the graph or not.

```suggestion
        f"{workflowhub_base_url}/workflows/{workflow_id}/ro_crate?version={workflow_version}/"
```